src/main/java/me/deadlight/ezchestshop/Listeners/PlayerTransactEvent.java

```diff
@@ -76,7 +77,7 @@ public int getCount() {
     public List<UUID> getAdminsUUID() {
         return this.admins;
     }
-    public Chest getChest() {
+    public Block getChest() {
```

Comment:
name change of getChest method

Reply:
Should we rename all the chest related variable names to something like shopblock or shopcontainer?